translations/sailfishos-chum-gui-de.ts

```diff
@@ -115,25 +115,25 @@
         <location filename="../qml/pages/PackagePage.qml" line="29"/>
         <source>Source code</source>
         <oldsource>Project Repository</oldsource>
-        <translation type="unfinished">Projekt Repository</translation>
+        <translation>Quellcode</translation>
```

Comment:
Sure? Quellcode is Sourcecode. I think it is better to translate it with 
"Projekt speicherort" or don´t translate it.

Reply:
"Quellcode" is mixed-language, "Quelltext" is the usual, but slightly antiquated German word for "source code".

Hence @black-sheep-dev please use "Quelltext" (or leave it untranslated as "Source code", but not as ~~"Sourecode"~~.

Reply:
I ultimately settled on "Quelltexte": It seems most natural as quite some Germans tend to say "Source codes" (in plural) when they use the English term.